sources/packages/forms/src/form-definitions/studentinformation.json

```diff
@@ -1295,7 +1536,7 @@
           "case": "",
           "truncateMultipleSpaces": false,
           "encrypted": false,
-          "redrawOn": "",
+          "redrawOn": "data",
```

Comment:
is redraw on data going to watch every property of data?

Reply:
Probably yes. This belongs to the address area that is not part of this PR. This type of `redrawOn` is also present across many other form.io definitions.
![image](https://user-images.githubusercontent.com/61259237/176011626-c8a5958e-8062-4cde-a71d-1d3f0b10fd89.png)